sponsorship/src/main/java/com/kyriba/conference/sponsorship/service/SponsorService.java

```diff
@@ -0,0 +1,23 @@
+package com.kyriba.conference.sponsorship.service;
+
+import com.kyriba.conference.sponsorship.api.dto.SponsorRegistrationRequest;
+import com.kyriba.conference.sponsorship.domain.Sponsor;
+import org.springframework.stereotype.Service;
+
+
+/**
+ * @author M-ASL
+ * @since v1.0
+ */
+@Service
+public class SponsorService

```

Comment:
Service is not needed at that stage.

Reply:
I believe, it is similar to https://github.com/gariklolkin/conference/pull/10#discussion_r286013126, so for now, I'd prefer to leave it as is